src/Stock.js

```diff
@@ -94,6 +97,9 @@ class Stock extends React.Component {
                     /> :
                     <h2>Invalid Ticker</h2>
                 }
+                 <div className = "stockInfo">
+                    <p>{this.state.stockTicker} Volume: {this.state.stockVolumeValues.at(0)}</p>
+                    </div>
```

Comment:
Can you fix the alignment of this </div> tag? Just so its in the same vertical position as the tag that opens it.

In fact, upon further inspection, the opening div tag is a little off as well. Proper indentation really helps people read the code. I think the opening tag needs 1 backspace, and then the closing one needs to be in line with it.

There's actually a tool to help with this. I believe that the default key bind is Ctrl+K -> Ctrl+F with the section highlighted. It should format all of the spacing correctly automatically.